test/cel-validation/securitypolicy_test.go

```diff
@@ -177,8 +177,8 @@ func TestSecurityPolicyTarget(t *testing.T) {
 							{
 								LocalPolicyTargetReference: gwapiv1a2.LocalPolicyTargetReference{
 									Group: gwapiv1a2.Group("gateway.networking.k8s.io"),
-									Kind:  gwapiv1a2.Kind("Gateway"),
-									Name:  gwapiv1a2.ObjectName("eg"),
+									Kind:  gwapiv1a2.Kind("HTTPRoute"),
```

Comment:
can we have one with `Gateway` so we also tst posiitive case

Reply:
Added in f1a1fd5262b6609b7617e39edee602514d3c4595